jit-compiler/src/util_code.rs

```diff
@@ -0,0 +1,54 @@
+use std::mem;
+
+use powdr_number::{FieldElement, KnownField};
+
+/// Returns the rust code containing utility functions for PIL types.
+pub fn util_code<T: FieldElement>() -> Result<String, String> {
+    if !(T::has_direct_repr() && (mem::size_of::<T>() == 8 || mem::size_of::<T>() == 4)) {
+        return Err(format!(
+            "Field {}not supported",
+            T::known_field()
+                .map(|f| format!("{f} "))
+                .unwrap_or_default()
```

Comment:
```suggestion
            "Field {} not supported",
            T::known_field()
                .map(|f| format!("{f}"))
                .unwrap_or_default()
```

Reply:
The idea is that if the field is not a known field, the error message says `"Field not supported"`

Reply:
ah ok